config.yaml

```diff
@@ -105,4 +105,10 @@ core_config_version: 0
 # argon2_hashing_pool_size:
 
 # (OPTIONAL | Default: "INFO"). Logging level for the core. Values are "DEBUG" | "INFO" | "WARN" | "ERROR" | "NONE"
-# log_level:
\ No newline at end of file
+# log_level:
+
+# (OPTIONAL | Default: null). The signing key used for firebase scrypt password hashing
+# firebase_signing_key:
```

Comment:
```suggestion
# firebase_password_hashing_signing_key:
```

Reply:
Also, is it a signing key for sure? Where does it say that in the firebase docs?

Reply:
In firebase's docs, the key is `base64_signer_key`, with the description as "The signer key from the project's password hash parameters"